app/src/pages/docs/index.module.sass

```diff
@@ -61,5 +63,16 @@
     &:nth-child(3)
       width: 60px
 
+  code
```

Comment:
After these changes `code` styles in the `pre` element were broken. Need to fix it.